### PR TITLE
[hi] Localize content/hi/docs/reference/glossary/etcd.md

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>
Fixes #35600 Localize content/en/docs/reference/glossary/etcd.md into content/hi/docs/reference/glossary/etcd.md

### CODE REVIEW

File: content/hi/docs/reference/glossary/etcd.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: इटीसीडी (etcd)
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/etcd.md:2]: This suggestion is based on the `ko` and `zh-cn` localization of the same file.
```suggestion
title: etcd
```

File: content/hi/docs/reference/glossary/etcd.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+full_link: /docs/tasks/administer-cluster/configure-upgrade-etcd/
+short_description: >
+  सभी क्लस्टर डेटा के लिए कुबेरनेट्स के समर्थन स्टोर के रूप में उपयोग किया जाने वाला लगातार और अत्यधिक उपलब्ध कुंजी मूल्य स्टोर।
+
+aka: 
+tags:
+- architecture
+- storage
+---
+ सभी क्लस्टर डेटा के लिए कुबेरनेट्स के समर्थन स्टोर के रूप में उपयोग किया जाने वाला लगातार और अत्यधिक उपलब्ध कुंजी मूल्य स्टोर।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/etcd.md:14]:
```suggestion
 सभी क्लस्टर डेटा के लिए कुबेरनेट्स के बैकिंग स्टोर के रूप में उपयोग किया जाने वाला सुसंगत और अत्यधिक उपलब्ध की-वैल्यू स्टोर।
```

File: content/hi/docs/reference/glossary/etcd.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+<!--more-->
+
+यदि आपका कुबेरनेट्स क्लस्टर अपने समर्थन स्टोर के रूप में इटीसीडी (etcd) का उपयोग करता है, तो सुनिश्चित करें कि आपके पास उन डेटा के लिए एक [बैक अप](/docs/tasks/administer-cluster/configure-upgrade-etcd/#backing-up-an-etcd-cluster) योजना है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/etcd.md:18]:
```suggestion
यदि आपका कुबेरनेट्स क्लस्टर बैकिंग स्टोर के रूप में etcd का उपयोग करता है, तो सुनिश्चित करें कि आपके पास उन डेटा के लिए एक [बैकअप](/docs/tasks/administer-cluster/configure-upgrade-etcd/#backing-up-an-etcd-cluster) योजना है।
```

File: content/hi/docs/reference/glossary/etcd.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+आप आधिकारिक [प्रलेखन](https://etcd.io/docs/) में आदि के बारे में गहन जानकारी पा सकते हैं।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/etcd.md:20]:
```suggestion
आप आधिकारिक [प्रलेखन](https://etcd.io/docs/) में etcd के बारे में गहन जानकारी प्राप्त कर सकते हैं।
```

File: content/hi/docs/reference/glossary/etcd.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+title: इटीसीडी (etcd)
+id: etcd
+date: 2018-04-12
+full_link: /docs/tasks/administer-cluster/configure-upgrade-etcd/
+short_description: >
+  सभी क्लस्टर डेटा के लिए कुबेरनेट्स के समर्थन स्टोर के रूप में उपयोग किया जाने वाला लगातार और अत्यधिक उपलब्ध कुंजी मूल्य स्टोर।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/etcd.md:7]:
```suggestion
  सभी क्लस्टर डेटा के लिए कुबेरनेट्स के बैकिंग स्टोर के रूप में उपयोग किया जाने वाला सुसंगत और अत्यधिक उपलब्ध की-वैल्यू स्टोर।
```

**Review by Babapool — COMMENTED** [review]
Some small nits from myside, hope you find them useful. Thanks

**Comment by divya-mohan0209** [comment]
@amolmote I recommend you sync your fork with the main repo before the change.

**Comment by amolmote** [comment]
> @amolmote I recommend you sync your fork with the main repo before the change.

sure @divya-mohan0209

**Comment by amolmote** [comment]
> Some small nits from myside, hope you find them useful. Thanks

thanks @Babapool , these are valid suggestions I will accept it.

**Pull request closed without merging**